HISTORY.md

```diff
@@ -0,0 +1,18 @@
+RELEASE HISTORY
```

Comment:
`HISTORY.md` -> `CHANGELOG.md`? Not important at all though.

Reply:
I've been on this fence on this myself. I think I spent a couple of hours poring over how other repos did the changelog and faq docs. `CHANGELOG` *would* be easier to file search for in GitHub global search